modules/ROOT/pages/platform/security.adoc

```diff
@@ -6,13 +6,29 @@
 label:AuraDB-Enterprise[]
 label:AuraDS-Enterprise[]
 
-AuraDB Enterprise and AuraDS Enterprise run within a Virtual Private Cloud (VPC) isolation for your deployment.
+AuraDB Enterprise and AuraDS Enterprise run in a dedicated cloud Account (AWS), Subscription (Azure) or Project (GCP) to achieve complete isolation for your deployment.
 
-The VPC enables you to operate within an isolated section of the service, where your processing, networking, and storage are further protected.
+Additional VPC boundaries enable you to operate within an isolated section of the service, where your processing, networking, and storage are further protected.
 
-Please note that the Aura Console runs in a separate VPC.
+Note that the Aura Console runs in a separate VPC, separate from the rest of Aura.
 
-=== AWS Private endpoints
+== Network Access
+
+Authorized users can configure certain parts of the infrastructure for running and handling Aura instances, and the networking used to establish secure connections between the database (Aura) and the application VPCs.
+This includes the ability to connect over the cloud provider's private link and private endpoint.
+
+You can also choose whether traffic to your Aura instance is allowed to traverse the public internet or instead uses the cloud provider's internal network. 
+To use the cloud provider's network, create a private endpoint inside your VPC. 
+This gives you a totally private connection to Aura.
+It's your choice if you want to prevent your Aura instances from being publicly accessible or if you want both public and private access.
```

Comment:
I suggest to rewrite this section to highlight the following:
* If you want your Aura instances to be private, do this.
* If you want your Aura instances to be both private and public, do this.
* If you want your Aura instances to be public, do this.

If the traffic is allowed to traverse the public internet, does that mean the instance is publicly accessible?
And if the traffic is routed through the cloud provider's network instead, is it then completely private? What do I need to do to allow both public and private access?

Reply:
@afharvey Hi Alex, please could I bring you in on this conversation to answer Jessica's questions?